src/cmap/connection.ts

```diff
@@ -149,7 +149,7 @@ export interface ConnectionOptions
 /** @public */
 export interface DestroyOptions {
   /** Force the destruction. */
-  force?: boolean;
+  force: boolean;
```

Comment:
@W-A-James Despite this and `CloseOptions` being marked `@public` they are not used on any publicly accessible APIs (connection.destroy and topology.close respectively). Normally, changing a public API wouldn't be viable in a fix PR like this but since these are not useful to users (they can't even be used as a type annotation for an input to one of our methods) I think this is an acceptable change to make here. 

Just sharing our thinking for other reviewers sake as well